recipe/meta.yaml

```diff
@@ -51,7 +51,7 @@ requirements:
   host:
     - python
     - setuptools
-    - cython
+    - cython =3
```

Comment:
I don't think we need `cython` for build ambertools.
You can try to remove it too see what happens.

Reply:
Nice, #122 seems to indicate it works fine!